src/planer.coffee

```diff
@@ -207,8 +207,8 @@ exports.processMarkedLines = (lines, markers, returnFlags = {}) ->
     return lines
 
   # Find inline replies (tm's following the first m in markers string)
-  inlineMatchRegex = new RegExp('m(?=e*((?:t+e*)+)m)', 'g')
-  while inlineReplyMatch = inlineMatchRegex.exec(lines)
+  inlineMatchRegex = new RegExp('m(?=e*(t[te]*)m)', 'g')
```

Comment:
I think it would be good to comment what this means. Even from the test i don't get why `t[te]` is used here.